packages/modals/src/elements/TooltipModal/TooltipModal.tsx

```diff
@@ -0,0 +1,205 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, {
+  useState,
+  useContext,
+  useMemo,
+  useEffect,
+  useRef,
+  HTMLAttributes,
+  ForwardRefExoticComponent,
+  PropsWithoutRef,
+  RefAttributes
+} from 'react';
+import PropTypes from 'prop-types';
+import { ThemeContext } from 'styled-components';
+import { usePopper, Modifier } from 'react-popper';
+import { useModal } from '@zendeskgarden/container-modal';
+import { useCombinedRefs } from '@zendeskgarden/container-utilities';
+import {
+  GARDEN_PLACEMENT,
+  getRtlPopperPlacement,
+  getPopperPlacement
+} from '../../utils/gardenPlacements';
+import { TooltipModalContext } from '../../utils/useTooltipModalContext';
+import {
+  StyledTooltipWrapper,
+  StyledTooltipModal,
+  StyledTooltipModalFooter,
+  StyledTooltipModalFooterItem,
+  StyledTooltipModalBackdrop
+} from '../../styled';
+import { Title } from './Title';
+import { Body } from './Body';
+import { Close } from './Close';
+
+interface IStaticTooltipModalExport<T, P>
+  extends ForwardRefExoticComponent<PropsWithoutRef<P> & RefAttributes<T>> {
+  Title: typeof Title;
+  Body: typeof Body;
+  Close: typeof Close;
+  Footer: typeof StyledTooltipModalFooter;
+  FooterItem: typeof StyledTooltipModalFooterItem;
+}
+
+export interface ITooltipModalProps extends HTMLAttributes<HTMLDivElement> {
+  /**
+   * The `HTMLElement` to position the `TooltipModal` relative to.
+   */
+  referenceElement?: HTMLElement;
+  /**
+   * [Popper.js v2 modifiers](https://popper.js.org/docs/v2/modifiers/)
+   * to customize positioning logic.
+   */
+  popperModifiers?: Array<Partial<Modifier<any, any>>>;
+  /**
+   * These placements differ from the default naming of Popper placements to help
+   * assist with RTL layouts.
+   **/
+  placement?: GARDEN_PLACEMENT;
+  hasArrow?: boolean;
```

Comment:
As mentioned, this component should expose `isAnimated` for Garden API consistency.